.github/workflows/ci.yaml

```diff
@@ -28,11 +41,15 @@ jobs:
           java-version: ${{ matrix.java_version }}
       - uses: ./src/fastddsgen/.github/actions/install-python-packages
       - uses: ./src/fastddsgen/.github/actions/fetch-fastddsgen-repos
+        with:
+          foonathan_memory_vendor_branch: '${{ github.event.inputs.foonathan_memory_vendor_branch }}'
+          fastcdr_branch: '${{ github.event.inputs.fastcdr_branch }}'
+          fastdds_branch: '${{ github.event.inputs.fastdds_branch }}'
 
       - name: Get minimum supported version of CMake
         uses: lukka/get-cmake@latest
         with:
-          cmakeVersion: 3.16.3
+          cmakeVersion: 3.22.6
```

Comment:
Why is the reason to upgrade the minimum CMake version? As far as I see, both Fast DDS and Fast CDR requires CMake 3.16.3 as its minimum version.

Reply:
Fast-CDR was upgraded to 3.22 and Fast-DDS will be also.